pkg/remote/aws/vpc_security_group_rule_enumerator.go

```diff
@@ -25,8 +25,8 @@ type securityGroupRule struct {
 	Type                  string
 	SecurityGroupId       string
 	Protocol              string
-	FromPort              int
-	ToPort                int
+	FromPort              float64
```

Comment:
Why a float ? A TCP|UDP port cannot be a floating point number

Reply:
As @moadibfr explained to me, we have to cast it as float64 because ctyjson deserialize integers as float64 and using int would mean to have a diff between serialized types and manual added types

Reply:
🆗